SDC_DB/createSchema.js

```diff
@@ -75,6 +75,6 @@ function write10Million(start = 1e7) {
     const command = 'mongoimport -d silverspoon -c restaurantmenus --file db/testData.json --type json --numInsertionWorkers 4';
```

Comment:
I added 18 workers you are being too modest lol - no change needed

Reply:
18?!? I need to get on that level